compiler/rustc_traits/src/chalk/db.rs

```diff
@@ -46,7 +46,9 @@ impl<'tcx> RustIrDatabase<'tcx> {
             .iter()
             .map(|(wc, _)| wc.subst(self.interner.tcx, bound_vars))
             .map(|wc| wc.fold_with(&mut regions_substitutor))
-            .filter_map(|wc| LowerInto::<Option<chalk_ir::QuantifiedWhereClause<RustInterner<'tcx>>>>::lower_into(wc, &self.interner)).collect()
+            .filter_map(|wc| LowerInto::<
+                    Option<chalk_ir::QuantifiedWhereClause<RustInterner<'tcx>>>
+                    >::lower_into(wc, &self.interner)).collect()
```

Comment:
Hmm, why did this formatting change...

Reply:
I changed this manually since it was well over 100 characters long.